### PR TITLE
minor changes of operations

### PR DESCRIPTION
**Pull request by RexWzh**

### CODE REVIEW

File: src/matrix.jl [hunk 1]
```diff
@@ -134,9 +134,10 @@ function makemask(matrix::AbstractArray, rule::Function)::BitArray{2}
     n = size(matrix, 1)
     mask = falses(size(matrix))
     for row in 1:n, col in 1:n
-        if isnothing(matrix[row, col]) && iszero(rule(row - 1, col - 1))
-            mask[row, col] = true
+        if isnothing(matrix[row, col]) && rule(row - 1, col - 1) == 0
+            @inbounds mask[row, col] = true
         end
+        # @inbounds mask[row, col] = isnothing(matrix[row, col]) && rule(row - 1, col - 1) == 0
```
Comment (johnnychen94) [inline src/matrix.jl:140]: remove this?

File: src/tables.jl [hunk 2]
```diff
@@ -236,8 +236,8 @@ Encode an integer into a `BitArray`.
 """
 function int2bitarray(k::Integer; pad::Int = 8)
     res = BitArray{1}(undef, pad)
-    @inbounds for i in pad:-1:1
-        res[i] = k & 1
+    for i in pad:-1:1
```
Comment (johnnychen94) [inline src/tables.jl:239]: The `@inbounds` macro works on the entire block, thus these are actually more-or-less equivalent.

**Review by johnnychen94 — APPROVED** [review]

File: src/matrix.jl (hunk 1 above)
Comment (RexWzh) [reply src/matrix.jl:140]: The comment is an equivalent way to the if-condition, but it is a bit slower. This is probably due to the assignment operation.

Comment (johnnychen94) [reply src/matrix.jl:140]: Unsure of it. This looks like a benchmark noise to me.

**Pull request merged**